atlp-rwanda/atlp-devpulse-fn
# #252 admin assign a cohort to a trainee

### PR Description
This PR allows admin to assign a cohort to a trainee

### Description of tasks that were expected to be completed

- As an admin i want to be able to assign a cohort to a trainee
- cohort has many trainees
- A trainee belongs to one cohort

### How has this been tested?
- Clone repo
- Checkout branch `ft-admin-assign-cohort-to-trainee`
- Run `npm install`, then `npm run dev` to check if the app is running successfully
- Login as admin and after login go to cohort page and view one cohort and add a trainee you want.

### Credentials
username: `admin@example.com`
password: `password123`

Issue link(#252 )

### Screenshots
![Screenshot 2024-11-15 104218](https://github.com/user-attachments/assets/2f4c5a44-52bc-48bf-b45c-e00010ab9f74)
![Screenshot 2024-11-15 104256](https://github.com/user-attachments/assets/0cea2083-6857-422d-8f90-5e1a0fb43ac4)
